README.md

```diff
@@ -1,19 +1,7 @@
-[![Crowdin](https://d322cqt584bo4o.cloudfront.net/translation-studio/localized.png)](https://crowdin.com/project/translation-studio)
-[![Build Status](https://travis-ci.org/unfoldingWord-dev/ts-desktop.svg?branch=develop)](https://travis-ci.org/unfoldingWord-dev/ts-desktop)
```

Comment:
we can change this to use our travis build if we want.

Reply:
I don't think it's worth the effort.  Maybe we can revisit it later.

Reply:
I already changed it.